.env

```diff
@@ -58,7 +58,7 @@ INFLUXDB_AUTH_TOKEN=
 # INFLUXDB_HOST=host.docker.internal
 
 # MongoDB (dev setting, using product opener network)
-MONGO_URI=mongodb://mongodb.po_default:27017
```

Comment:
can we keep that as a comment for those who have a local product opener instance ?